engine/pkg/meta/connection.go

```diff
@@ -21,19 +21,18 @@ import (
 
 // NewClientConn new a client connection
 func NewClientConn(storeConf *metaModel.StoreConfig) (metaModel.ClientConn, error) {
-	var cc metaModel.ClientConn
+	var (
+		cc  metaModel.ClientConn
+		err error
+	)
 
 	switch storeConf.StoreType {
 	case metaModel.StoreTypeEtcd:
-		cc = etcdkv.NewClientConnImpl()
+		cc, err = etcdkv.NewClientConnImpl(storeConf)
```

Comment:
```suggestion
		return etcdkv.NewClientConnImpl(storeConf)
```
Maybe it's a little bit clearer?